### PR TITLE
[bugfix/FARM-509] User should not be able to remove the last profile

### PR DESCRIPTION
**Pull request by davidfranquet**
[FARM-509]

#### 📲 What

User should not be able to remove the last profile

#### 🤔 Why
		
Needed for MVP

#### 👀 See
		
ezgif com-video-to-gif-15

		 
#### ✅ Acceptance criteria

- [ ] Design Review for UI with BA completed. 
- [ ] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [ ] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [ ] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [ ] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

@farmsmart/amido @farmsmart/wamf

### CODE REVIEW

**Review by alexanishchenko88 — DISMISSED** [review]

File: lib/ui/profile/Profile.dart [hunk 1]
```diff
@@ -74,7 +74,8 @@ class _Strings {
       'https://sites.google.com/farmsmart.co/farmsmart/home/privacy-policy?authuser=0';
   static final termsOfUseUrl =
       'https://sites.google.com/farmsmart.co/farmsmart/home/privacy-policy?authuser=0';
-  static final shareLink = 'https://play.google.com/store/apps/details?id=co.farmsmart.app';
+  static final shareLink =
+      'https://play.google.com/store/apps/details?id=co.farmsmart.app';
```
Comment (albertmiro) [inline lib/ui/profile/Profile.dart:78]: That has been done in another PR, isn't it?

Comment (davidfranquet) [reply lib/ui/profile/Profile.dart:78]: yep It's just format code

**Review by alexanishchenko88 — APPROVED** [review]

**Review by albertmiro — APPROVED** [review]

**Review by LeeMatthewHiggins — APPROVED** [review]
